src/go/api/scorch/app.go

```diff
@@ -326,6 +332,38 @@ func (this Scorch) stopFilebeat(ctx context.Context, cmd *exec.Cmd, port int) {
 	}
 }
 
+func (this Scorch) recordInfo(runID int, runDir string, name string, startTime time.Time, versionCommit string, versionTag string, versionDate string) error {
+	// runID, runDir, exp.Metadata.Name, start, version.Commit, version.Tag version.Date
+
+	var (
+		c    *mmcli.Command
+		resp string
+		err  error
+		info string
+	)
+
+	c = mmcli.NewCommand()
+	c.Command = "version"
+	if resp, err = mmcli.SingleResponse(mmcli.Run(c)); err != nil {
+		return fmt.Errorf("running minimega verssion %w", err)
+	}
+
+	info = fmt.Sprintf("Experiment Name: %s\nExperiment Commit Hash: pending\nScorch Run Name: %s\nStart Time: %v\nEnd Time: %v\nPhenix Version: %s %s %s \nMinimega Version: %s",
+		name, this.md.RunName(runID), startTime, time.Now().UTC(), versionCommit, versionTag, versionDate, resp)
+
+	if _, err = os.Stat(runDir); os.IsNotExist(err) {
+		if err = os.MkdirAll(runDir, 0755); err != nil {
+			return fmt.Errorf("creating %s directory for scorch run %d: %w", runDir, runID, err)
+		}
+	}
+
+	if err := os.WriteFile(filepath.Join(runDir, fmt.Sprintf("info-scorch-run-%d_%s.txt", runID, startTime.Format(time.RFC3339))), []byte(info), 0666); err != nil {
```

Comment:
@nblair2 since you use `fmt.Sprintf("info-scorch-run-%d_%s.txt", runID, startTime)` here and in the error message below, you might consider just pulling it out into its own variable so that 1) the formatted string only has to be generated once, and 2) readability is a little bit better.